earthfile2llb/interpreter.go

```diff
@@ -1433,6 +1436,34 @@ func (i *Interpreter) handleArg(ctx context.Context, cmd spec.Command) error {
 	return nil
 }
 
+func (i *Interpreter) handleLet(ctx context.Context, cmd spec.Command) error {
+	if i.pushOnlyAllowed {
+		return i.pushOnlyErr(cmd.SourceLocation)
+	}
+	var opts letOpts
+	argsCpy := getArgsCopy(cmd)
+	args, err := parseArgs("LET", &opts, argsCpy)
+	if err != nil {
+		return errors.Wrap(err, "failed to parse SET args")
+	}
+	if len(args) != 3 || args[1] != "=" {
+		return hint.Wrap(errInvalidSyntax, "LET requires a variable assignment, e.g. LET foo = bar")
+	}
```

Comment:
I'm just wondering what the reasoning is for disallowing a declaration without setting a value.
Since `LET` is pretty much a local, mutable, variable declaration it follows that one might expect `LET foo` to not through an error and perhaps default to empty.